app/views/content/teacher-training-advisers.md

```diff
@@ -49,68 +43,88 @@ youtube_video:
   about-teacher-training-advisers-experience-video:
     id: T9Bhcaa6LJ4
     title: A video about what teaching experience our Teacher Training Advisers have
+content:
+    - content/teacher-training-advisers/blog_links
 ---
-Teacher training advisers give you free, one-to-one help and support. 
-
-They're all former teachers with lots of experience, so they know the process inside out.
+Teacher training advisers give you free, one-to-one support, wherever you are on your journey into teaching.
 
-## What an adviser can do for you
+They have years of teaching experience and can answer all your questions about life in the classroom, as well as how you can get there.
 
 $q-tta-one$
 
-An adviser can give you as much or as little help as you need. 
+## How does it work?
 
-Other trainee teachers say they are particularly helpful with: 
+You’ll be matched with your own dedicated adviser. You’ll have the same adviser throughout your journey into teaching.
+How you talk to them is up to you – you can chat through:
 
-* arranging school experience
-* seeking out financial support
-* writing a personal statement
-* preparing for interviews
+* email
+* phone
+* text
 
-## Meet Emma, a teacher training adviser
+You can talk to them as little or as often as you need. This could be every week for a year, or just a few times before making an application.
 
-$about-teacher-training-advisers-video$
+## How your adviser can help you
 
-## Helping you decide if teaching is right for you
+### If teaching is just an idea
 
 $q-tta-two$
 
-Teaching is an exciting career, but if you have caring responsibilities, people dependent on your earnings, or you are living with a disability you might need some help to take your next step. 
+Your adviser can talk to you about what teaching is really like and help you understand if it’s right for you.
+
+They can:
+
+* talk through your questions and any doubts ​
+* help you find school experience
+* explain the steps to becoming a teacher
+
+### If you're ready to apply
+
+Your adviser can talk you through the application process step-by-step.
 
-Your adviser can help you navigate your path into teaching, and make sure you get any additional help that is available. Your adviser will be just a call, message or email away.
+They can help you:
 
-## Meet Simon, a teacher training adviser
+* write your personal statement
+* understand your different training options
+* prepare for interviews
 
-$about-teacher-training-advisers-experience-video$
+### If you've applied and been unsuccessful
 
-## Already applied and been unsuccessful?
+$q-tta-three$
```

Comment:
Is this quote in the right place? Feels more like it belongs in the section before?